### PR TITLE
@task PDA-1203 UI - Add PARASOFT JMS/MQ PROXY section on demo admin page

### PR DESCRIPTION
**Pull request by rliu-parasoft**

### CODE REVIEW

File: src/main/resources/static/aerospace/css/common.css [hunk 1]
```diff
@@ -121,4 +121,8 @@ body {
 
 .home-link:hover {
     color: #E7FFA1;
+}
+
+.info-icon {
+    color: #E7FFA1;
```
Comment (rliu-parasoft) [inline src/main/resources/static/aerospace/css/common.css:127]: Currently, I set its base color same as 
image

File: src/main/resources/static/common/css/common.css [hunk 2]
```diff
@@ -406,3 +406,9 @@ a, a:hover, a:focus {
 .regions {
     max-height: 28px;
 }
+
+.info-icon {
+    font-size: initial;
+    color: #0093d2;
+    vertical-align: text-top;
+}
```
Comment (rliu-parasoft) [inline src/main/resources/static/common/css/common.css:414]: It works in both industries Defense and Outdoor

Defense:
image

Outdoor:
image

Comment (yshi-parasoft) [reply src/main/resources/static/common/css/common.css:414]: Maybe change this color to be a darker one which is consistent with the "DemoApp OpenAPI" text: #0366AE.

Comment (yshi-parasoft) [reply src/main/resources/static/common/css/common.css:414]: Also please put the icon in the middle to be aligned with the text.

**Pull request merged**